inttest/Makefile

```diff
@@ -30,7 +30,9 @@ bin/sonobuoy: | bin
 	  --build-arg ETCD_ARCH=$(etcd_arch) \
 	  --build-arg ETCD_VERSION=$(etcd_version) \
 	  --build-arg KUBE_VERSION=$(kubernetes_version) \
-	  -t footloose-alpine - <'$<'
+	  -t footloose-alpine \
+	  -f footloose-alpine/Dockerfile \
+	  footloose-alpine
```

Comment:
Do we need the build context here?

Reply:
build context is  the last arg `footloose-alpine`

Reply:
Yes, but do we need it?

Reply:
yes, we need it as the footloose-alpine Dockerfile adds files from `footloose-alpine` dir (a.k.a build context)

Reply:
Oops, I overlooked cri-dockerd.sh... :blush: